### DISCUSSION TITLE
depremyardim.com üzerinden yapılan ihbarların saat kaçta yapıldığının gösterilmesi

### DISCUSSION THREAD

**Discussion by b-tekinli** [acikkaynak/deprem-yardim-projesi#49] [Feedback] [answered]
depremyardim.com üzerinden yapılan ihbarların saat kaçta ve hangi gün yapıldığının gösterilmesi gerektiğini düşünüyorum. aynı zamanda filtreleme yaparken de mesela son 1 saat içinde yapılmış ihbarları vs. görmek istiyorum çok işe yarıyor. arayarak ihbar yapıyorum teyit etmek için ya da ekiplerle direkt olarak iletişim kurmak için ama kaç saat önce o enkazdan ses gelmiş bu infoları veremiyorum sıkıntı oluyor.

**Comment by ramazansancar** [comment, marked as answer]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebinizi [bu](https://github.com/acikkaynak/deprem-yardim-com) reponun issues veya pr alanına taşıyabilirsiniz.
